src/clj/rems/db/user_settings.clj

```diff
@@ -0,0 +1,21 @@
+(ns rems.db.user-settings
+  (:require [rems.db.core :as db]
+            [clojure.tools.logging :as log]
+            [rems.json :as json]))
+
+(defn update-user-settings! [user settings]
+  (assert (and user settings) "User or user settings missing!")
+  (db/update-user-settings!
+   {:user user
+    :settings (json/generate-string {:language (:language settings)})})
+  {:success true})
+
+(defn- parse-settings [settings]
+  (let [{:keys [language]} (json/parse-string settings)]
+    {:language (keyword language)}))
```

Comment:
Since :language is an optional key in UpdateUserSettingsCommand, how does this handle nil language? We probably need a test which sets empty settings and then reads them back.

Reply:
Done, robustified both get-user-settings and update-user-settings! and added a couple of tests.